StorageManagementSystem/Client/test/Test_Client_StorageManagementSystem.py

```diff
@@ -37,6 +37,10 @@
                                                     'Failed':{}} )
 mockObjectSE5.getStatus.return_value = S_OK( {'DiskSE': True, 'TapeSE':False} )
 
+mockObjectDMSHelper = MagicMock()
+mockObjectDMSHelper.getLocalSiteForSE.return_value = S_OK( 'mySite' )
+mockObjectDMSHelper.getSitesForSE.return_value = S_OK( ['mySite'] )
```

Comment:
yes, when selecting the best site for staging we try and choose SEs at the same site (this is new)